### PR TITLE
tools, media: add sync-diff-inspector guide

### PR DESCRIPTION
**Pull request by lilin90**
@WangXiangUSTC @CaitinChen PTAL
Via: https://github.com/pingcap/docs-cn/pull/935

I see that currently this guide hasn't been added to the Readme TOC. Do we need to add it to README and make it display on PingCAP website? @WangXiangUSTC

### CODE REVIEW

File: tools/sync-diff-inspector.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+# sync-diff-inspector User Guide
+
+[sync-diff-inspector](https://github.com/pingcap/tidb-tools/tree/master/sync_diff_inspector) is a tool used to check whether the data in MySQL is consistent with that in TiDB. In addition, you can also use this tool to repair data in the scenario of a small number of data is inconsistent.
+
+This guide introduces the key features of sync-diff-inspector and describes how to configure and use this tool. You can download it at [sync-diff-inspector-linux-amd64.tar.gz](https://download.pingcap.org/sync-diff-inspector-linux-amd64.tar.gz).
+
+## Key features
+
+- Compare the table structure and data
```
Comment (CaitinChen) [inline tools/sync-diff-inspector.md]:
```suggestion
- Compare the table schema and data
```

File: tools/sync-diff-inspector.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+- Generate the SQL statements used to repair data if the data inconsistency exists
+- Support comparing the data of multiple tables with the data of a single table (for the scenario of synchronizing data from sharded tables into the combined table)
+- Support comparing the data of different schemas or tables with different names
```
Comment (CaitinChen) [inline tools/sync-diff-inspector.md]:
```suggestion
- Support comparing the data of different databases or tables with different names
```

File: tools/sync-diff-inspector.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+# Whether to compare data using the implicit column "_tidb_rowid".
+# You can enable it when the compared tables have no primary or unique key,
+# and the two compared databases are both TiDB.
+use-rowid = false
+
+# If enabled, the chunk's checksum is calculated and data is compared by checksum.
+# If disabled, data is compared line by line.
+use-checksum = true
+
+# If set true, data check is ignored. If set false, data is checked.
```
Comment (CaitinChen) [inline tools/sync-diff-inspector.md]:
```suggestion
# If set to true, data check is ignored. If set to false, data is checked.
```

File: tools/sync-diff-inspector.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+ignore-data-check = false
+
+# If set true, the table struct comparison is ignored.
```
Comment (CaitinChen) [inline tools/sync-diff-inspector.md]: ditto

File: tools/sync-diff-inspector.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+# If set false, the table struct is compared.
```
Comment (CaitinChen) [inline tools/sync-diff-inspector.md]: ditto

File: tools/sync-diff-inspector.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+# Ignores checking some columns. But these columns can still be used to
+# divide chunks and sort the checked data.
+# ignore-columns = ["name"]
+
+# Removes some columns. During checking, these columns are removed from the table struct.
+# These columns are neither checked nor used to divide chunks or sort data.
+# remove-columns = ["name"]
+
+# Configuration example of comparing two tables with different database name and table name.
```
Comment (CaitinChen) [inline tools/sync-diff-inspector.md]:
```suggestion
# Configuration example of comparing two tables with different database names and table names.
```

**Comment by WangXiangUSTC** [comment]
I will add this file's link in other tool's @lilin90

**Comment by lilin90** [comment]
@WangXiangUSTC Sounds good. I've updated the description. Please have another review and check whether that's what you mean. Thanks!

**Comment by lilin90** [comment]
@CaitinChen @WangXiangUSTC PTAL again

**Comment by lilin90** [comment]
@WangXiangUSTC PTAL again

File: tools/sync-diff-inspector.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
+---
+title: sync-diff-inspector User Guide
+summary: Use sync-diff-inspector to compare data and repair inconsistent data.
+category: tools
+---
+
+# sync-diff-inspector User Guide
+
+[sync-diff-inspector](https://github.com/pingcap/tidb-tools/tree/master/sync_diff_inspector) is a tool used to compare the data that is stored in databases with the MySQL protocol. For example, it can compare the data in MySQL with that in TiDB, the data in MySQL with that in MySQL, or the data in TiDB with that in TiDB. In addition, you can also use this tool to repair data in the scenario where a small amount of data is inconsistent. In addition, you can also use this tool to repair data in the scenario where a small amount of data is inconsistent.
```
Comment (WangXiangUSTC) [inline tools/sync-diff-inspector.md]: double `In addition, you can also use this tool to repair data in the scenario where a small amount of data is inconsistent.`, can remove one

**Comment by lilin90** [comment]
@WangXiangUSTC Addressed. Thanks!

**Pull request merged**